doc/reference/tooling/tt_cli/import.rst

```diff
@@ -0,0 +1,279 @@
+    In this case, field values start from the second line.
+
+    See also: :ref:`Matching of input and space fields <tt-import-match-fields>`.
+
+..  option:: --log STRING
+
+    Specify the name of a log file containing information about import errors (the default is ``import``).
+    If the log file already exists, new data is written to this file.
+
+..  option:: --match STRING
+
+    Configure matching between field names in the input file and the target space.
+
+    See also: :ref:`Matching of input and space fields <tt-import-match-fields>`.
+
+..  option:: --null STRING
+
+    Set a value to be interpreted as ``null`` when importing data.
+    By default, an empty value is interpreted as ``null``.
+    For example, a tuple imported from the following row ...
+
+    .. code-block:: text
+
+        1,477,Andrew,,38
+
+    ... should look as follows: ``[1, 477, 'Andrew', null, 38]``.
+
+..  option:: --on-error STRING
+
+    Define an action performed if a row to be imported cannot be parsed correctly.
+    Possible values:
+
+    *   ``stop``: stop importing data.
+    *   ``skip``: skip rows whose data cannot be parsed correctly.
+
+    Duplicate primary key errors are handled using the ``--on-exist`` option.
+
+    See also: :ref:`Handling parsing errors <tt-import-parsing-error>`.
+
+..  option:: --on-exist STRING
+
+    Define an action performed if a duplicate primary key error occurs.
+    Possible values:
+
+    *   ``stop``: stop importing data.
+    *   ``skip``: skip existing values when importing.
+    *   ``replace``: replace existing values when importing.
+
+    Other errors are handled using the ``--on-error`` option.
+
+    See also: :ref:`Handling duplicate primary key errors <tt-import-duplicate-error>`.
+
+..  option:: --password STRING
+
+    Specify a password used to connect to the instance.
+
+..  option:: --progress STRING
+
+    Specify the name of a progress file that stores the following information:
+
+    *   The positions of lines that were not imported at the last launch.
+    *   The last position that was processed at the last launch.
+
+    If a file with the specified name exists, it is taken into account when importing data.
+    ``tt import`` tries to insert lines that were not imported and then continues importing from the last position.
+
+    At each launch, the content of a progress file with the specified name is overwritten.
+    If the file with the specified name does not exist, a progress file is created with the results of this run.
+
+    .. NOTE::
+
+        If the option is not set, then this mechanism is not used.
+
+..  option:: --quote STRING
+
+    Set the symbol that defines a quote.
+    For CSV, double quotes are used by default (``"``).
+    The double symbol of this option acts as the escaping symbol within input data.
+
+..  option:: -success STRING
+
+    The name of a file with rows that were imported (the default is ``success``).
+    Overrides the file if it already exists.
```

Comment:
Can't understand what exactly this means. Overwrites?

Reply:
exactly